virt/virtual_machines/virtual_disks/virt-vm-disk-enlargment.adoc

```diff
@@ -0,0 +1,16 @@
+[id="virt-vm-disk-enlargment"]
```

Comment:
Please add the newly required content type above this line (applies to the modules as well). See https://docs.google.com/document/d/1o4rrsgowUzGiVSH84Wg-Tt4DxmDXE_hg2H946NnXj6s/edit?usp=sharing for more details